src/fstext/context-fst-inl.h

```diff
@@ -310,9 +310,9 @@ bool ContextFstImpl<Arc, LabelT>::CreateArc(StateId s,
     // the output arcs, just 0.
     return CreatePhoneOrEpsArc(s, nextstate, olabel, phoneseq, oarc);
   } else {
-    std::cerr << "ContextFst: CreateArc, invalid olabel supplied [confusion about phone list or disambig symbols?]: "<<(olabel);
-    exit(1);
- }
+    KALDI_ERR << "ContextFst: CreateArc, invalid olabel supplied [confusion "
```

Comment:
fine.

On Fri, Apr 15, 2016 at 12:58 PM, Kirill Katsnelson <
notifications@github.com> wrote:

> In src/fstext/context-fst-inl.h
> <https://github.com/kaldi-asr/kaldi/pull/704#discussion_r59932486>:
>
> > @@ -310,9 +310,9 @@ bool ContextFstImpl<Arc, LabelT>::CreateArc(StateId s,
> >      // the output arcs, just 0.
> >      return CreatePhoneOrEpsArc(s, nextstate, olabel, phoneseq, oarc);
> >    } else {
> > -    std::cerr << "ContextFst: CreateArc, invalid olabel supplied [confusion about phone list or disambig symbols?]: "<<(olabel);
> > -    exit(1);
> > - }
> > +    KALDI_ERR << "ContextFst: CreateArc, invalid olabel supplied [confusion "
>
> Ok, sounds good. I'll push to this PR unless you think a separate one is
> better.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/704/files/2d2427aa2d4a7932b747471ff663a41be67671ae#r59932486>
>